admin/services/restart.service.js

```diff
@@ -86,7 +86,7 @@ restartService.restart = async (
     throw new Error(`One of the pupils is not eligible for a restart`)
   }
   const restartReasonId = await pupilRestartDataService.sqlFindRestartReasonByCode(restartReasonCode)
-  return Promise.all(pupilsList.map(async pupilId => {
+  await Promise.all(pupilsList.map(async pupilId => {
     const pupilRestartData = {
```

Comment:
Outside of the scope of this, just noticed this doesn't looks quite good.
We'll need to refactor this into a multiple insertion statement.